packages/backend/src/graphql/resolvers/payment_pointer.test.ts

```diff
@@ -272,6 +272,7 @@ describe('Payment Pointer Resolvers', (): void => {
       assert.ok(updatedPaymentPointer)
 
       const {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I added the ignore rule because I am destructuring `deactivatesAt` and then not using it on purpose to make a new object from `paymentPointer` without that property. I thought maybe `deactivatesAt: _` would work but it doesnt. 

I preferred this over refactoring to copy and then delete properties. I think this is just a case where we actually dont want the rule to apply so I added the ignore.